cmd/placement/cmd/placement.go

```diff
@@ -122,7 +130,24 @@ func Run(ctx context.Context, options *placementoptions.Options) error {
 		resyncPeriod,
 	)
 
-	// create the kcp-scheduling-placement-controller
+	// create edgeSharedInformerFactory
+	providerConfig, _ := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{},
+	).ClientConfig()
+	edgeViewConfig, err := configForViewOfExport(ctx, providerConfig, "edge.kcp.io")
+	if err != nil {
+		logger.Error(err, "failed to create config for view of edge exports")
+		return err
+	}
+	edgeClusterClientset, err := edgeclientset.NewForConfig(edgeViewConfig)
+	if err != nil {
+		logger.Error(err, "failed to create clientset for view of edge exports")
+		return err
+	}
+	edgeSharedInformerFactory := edgeinformers.NewSharedInformerFactoryWithOptions(edgeClusterClientset, resyncPeriod)
```

Comment:
You might want to put "cluster" in the name of this factory, to make its scope explicit.
Or you might not.

Reply:
Same as above.